helm/reana/values.yaml

```diff
@@ -60,7 +60,6 @@ components:
     image: reanahub/reana-server:0.8.0-alpha.1
     environment:
       REANA_MAX_CONCURRENT_BATCH_WORKFLOWS: 30
-      REANA_USER_EMAIL_CONFIRMATION: true
```

Comment:
Can you explain why this was removed? I'm failing to understand, but I guess the default values should be also modified [here](https://github.com/reanahub/reana/blame/2c5df09dc9f6fa02e0321330ec79af9bbe480a0d/helm/reana/README.md#L15)

Reply:
It was accidental. Fixed.